package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -24,6 +30,11 @@ def get_db(session_class: sessionmaker) -> Generator:
         database.close()
 
 
+def _get_dbname():
+    username = os.environ.get("KEDRO_SQLITE_STORE_USERNAME") or getpass.getuser()
+    return username + ".db"
```

Comment:
I have to do it this way rather than a single return statement because if do that the related unit test fails (gives me None.db)

Reply:
Let's talk about this!

Reply:
There are chances you have already setup an env variable on your machine and if so the related unit test will fail. So we have to explicitly mock the environment variable for the tests -- see below. 

![image](https://github.com/kedro-org/kedro-viz/assets/37628668/fc8a1bc8-2214-4c64-a372-f6913fa46620)

Now for second test when we mock env variable to None, weirdly with the one-liner, it returns None.db instead of doing getpass.getUser() and returning the computer_name.db 

Hence I reverted to this just so it passes the unit tests.